litestar/dto/factory/_backends/abc.py

```diff
@@ -162,6 +154,9 @@ def parse_model(
             if should_ignore_field(field_definition, self.context.dto_for):
                 continue
 
+            if should_mark_private(field_definition, self.context.config.underscore_fields_private):
+                field_definition.dto_field.mark = Mark.PRIVATE
```

Comment:
I think `Mark` is not a good name. 

Additionally, instead of an enum - I'd use a sentinel class for this.

Reply:
Well that's a little beyond the scope of this PR. It has been called `Mark` since the very first DTO PR.

Reply:
i like the `Mark` name tho.